src/scripting/lua_root.cc

```diff
@@ -1223,11 +1223,15 @@ void LuaDescriptions::do_modify_productionsite(lua_State* L,
 	}
 }
 
-void LuaDescriptions::do_modify_ship(lua_State* L, const std::string&, const std::string&) {
+void LuaDescriptions::do_modify_ship(lua_State* L,
```

Comment:
Why don't we need to /NOLINT this instance of lua_State* L like in the multiple instances above